tools/topology/topology1/development/sof-imx8mp-compr-aac-wm8960.m4

```diff
@@ -0,0 +1,98 @@
+#
+# Topology with codec_adapter processing component for i.MX8QM/i.MX8QXP
```

Comment:
Can you, please, update the target name for both 8MP topologies.

Reply:
sure.

Reply:
should be fine now.

Reply:
It is! Thanks!